go/cmd/dolt/commands/engine/sqlengine.go

```diff
@@ -53,7 +55,10 @@ func NewSqlEngine(
 	format PrintResultFormat,
 	initialDb string,
 	isReadOnly bool,
-	tempUsers []gms.TemporaryUser,
+	mysqlDbFilePath string,
```

Comment:
This method now officially has too many params. Time to introduce a config struct to hold them.

You can do that on a second pass after you get this in